docs/content/meta/release-notes.md

```diff
@@ -11,6 +11,67 @@ title: Release Notes
 weight: 10
 ---
 
+## **0.16.0** TBD
+
+Hugo now has:
+
+* 8700 (+2000) stars on GitHub
+* 284 (+50) contributors
+* 77 (+10) themes
+
+### New Features
+
+* Add emoji support {{<gh 1892>}}
+* Add `themesDir` option to configuration {{<gh 1556>}}
+* Add support for Go 1.6 `block` keyword in templates {{<gh 1832>}}
+* Partial static sync {{<gh 1644>}}
+* Source file based relative linking (a la Github) {{<gh 0x0f6b334b6715253b030c4e783b88e911b6e53e56>}}
```

Comment:
I'm doing some of the cleanup on this atm - and will add some docs.

its worth noting that right now, its a blackfriday only feature.